assets/css/_dialogs.scss

```diff
@@ -170,6 +199,78 @@ $dialog-box-default-width: 790px;
   text-align: center;
 }
 
+.map-dialog {
+  width: 100vh;
+  padding: 0;
+}
+
+button.close {
```

Comment:
This selector might be a little too generic, since it applies to any `<button class="close">` -- and those are everywhere. We should be a little more specific about which button this style overrides.